src/components/standalone/monitoring/ConnectivityMonitor.vue

```diff
@@ -15,13 +16,17 @@ import {
 import { computed, onMounted, ref } from 'vue'
 import { useI18n } from 'vue-i18n'
 import WanEventsCard from './connectivity/WanEventsCard.vue'
-import InterfaceTrafficCard from './connectivity/InterfaceTrafficCard.vue'
 import { isEmpty } from 'lodash-es'
 import type { Policy } from '@/composables/useMwan'
 import WanConnectionsCard from './connectivity/WanConnectionsCard.vue'
 import { useNetworkDevices } from '@/composables/useNetworkDevices'
 import { getIpv4Addresses, getIpv6Addresses, getName, isDeviceUp } from '@/lib/standalone/network'
 import { useUciNetworkConfig } from '@/composables/useUciNetworkConfig'
+import InterfaceTrafficCard from './connectivity/InterfaceTrafficCard.vue'
```

Comment:
We have different imports all over the place, some with `.` some with `@`. Maybe in the future it's a good thing to make them all coming either from `.` or `@`.